src/rattler_build_conda_compat/loader.py

```diff
@@ -164,3 +165,39 @@ def load_all_tests(content: dict[str, Any]) -> list[dict]:
         evaluated_tests.extend(list(visit_conditional_list(section)))
 
     return evaluated_tests
+
+def jinja_env() -> jinja2.Environment:
+    # Create a custom Jinja2 environment with modified syntax
+    return jinja2.Environment(
+        variable_start_string="${{",
+        variable_end_string="}}",
+        trim_blocks=True,
+        lstrip_blocks=True,
+        autoescape=True,
+    )
+
+def render_context(context: dict[str, str]) -> dict[str, str]:
+    env = jinja_env()
+    # Process each key-value pair in the dictionary
+    for key, value in context.items():
+        # If the value is a string, render it as a template
+        if isinstance(value, str):
+            template = env.from_string(value)
+            rendered_value = template.render(context)
+            context[key] = rendered_value
+
+    return context
+
+
+def eval_jinja(content: dict[str, Any]) -> dict[str, Any]:
```

Comment:
I think it's better to pass env explicityl as an argument, so it will be used to render the things